### PR TITLE
Add last calendar ranges to quick ranges

### PR DESCRIPTION
**Pull request by lasanthaS**
## Purpose

The Date range picker widget, only contains fixed time period ranges for quick ranges to select periods from `current time - n`. With the introduction of persisted aggregation, this has been changed and modified to select only calendar units (`1 day` means `yesterday`). 

This was misleading and hence introduced another set of quick ranges to select both of the above.

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/resources/widgetConf.json [hunk 1]
```diff
@@ -70,4 +68,4 @@
       }
     ]
   }
-}
+}
```
Comment (AnuGayan) [inline components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/resources/widgetConf.json:71]:
```suggestion
}

```

File: components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/src/widget/DateTimePicker.jsx [hunk 2]
```diff
@@ -1090,4 +1252,4 @@ class DateTimePicker extends Widget {
   };
 }
 
-global.dashboard.registerWidget('DateRangePicker', DateTimePicker);
+global.dashboard.registerWidget('DateRangePicker', DateTimePicker);
```
Comment (AnuGayan) [inline components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/src/widget/DateTimePicker.jsx:1255]:
```suggestion
global.dashboard.registerWidget('DateRangePicker', DateTimePicker);

```

File: components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/resources/widgetConf.json (hunk 1 above)
Comment (lasanthaS) [reply components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/resources/widgetConf.json:71]: Fixed

File: components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/src/widget/DateTimePicker.jsx (hunk 2 above)
Comment (lasanthaS) [reply components/org.wso2.analytics.apim.widgets/DateTimeRangePicker/src/widget/DateTimePicker.jsx]: Fixed

**Review by AnuGayan — APPROVED** [review]

**Pull request merged**